Sage/carbon
# fix(pages): maintain height of content when page slides in

Currently, when moving from a page with a small amount of content to one with more content, the height of the new page content is truncated (with a scrollbar) and suddenly "jumps" to the new height when the transition is completed - a rather unexpected effect. This is due to the absolute position on the page content as it slides in, which stops its height being taken into account when calculating the parent's height. The fix is to remove this absolute positioning (moving it up to the page wrapper so that the slide effect still works).

fix #6492

### Proposed behaviour



When a new page slides in which is taller than the previous one, it should take up the full space it needs while sliding.

### Current behaviour

The new page is truncated to the height of previous page while sliding, suddenly "jumping" to its full height only when the slide has finished



### Checklist




#### QA


### Additional context



### Testing instructions



The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.


https://codesandbox.io/s/pages-scroll-forked-c22mv3
